src/main/java/com/flink/connector/BigQueryDynamicTableSource.java

```diff
@@ -0,0 +1,61 @@
+package com.flink.connector;
+
+import org.apache.flink.api.common.serialization.DeserializationSchema;
+import org.apache.flink.streaming.api.functions.source.SourceFunction;
+import org.apache.flink.table.connector.ChangelogMode;
+import org.apache.flink.table.connector.format.DecodingFormat;
+import org.apache.flink.table.connector.source.DynamicTableSource;
+import org.apache.flink.table.connector.source.ScanTableSource;
+import org.apache.flink.table.connector.source.SourceFunctionProvider;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.types.DataType;
+
+
+public final class BigQueryDynamicTableSource implements ScanTableSource {
+	private static final long serialVersionUID = 1;
+	private final String project_id;
+	private final String dataset;
+	private final String table;
+	private final int snapshotMillis;
+	private final DecodingFormat<DeserializationSchema<RowData>> decodingFormat;
+	private final DataType producedDataType;
+	public BigQueryDynamicTableSource(String project_id,String dataset,String table,int snapshotMillis, DecodingFormat<DeserializationSchema<RowData>> decodingFormat,
```

Comment:
keep java naming conversions, should be projectId (see https://google.github.io/styleguide/javaguide.html)

Reply:
Modified field names as per standards